internal/pkg/values/dayTime.go

```diff
@@ -38,7 +38,7 @@ func parseDayTime(s string) (*dayTime, error) {
 	}
 
 	if minute < 0 || minute >= 60 {
-		return nil, newInvalidConfigError("time of day")
+		return nil, newInvalidSyntaxError("time of day", "minutes must be in a range from 0 to 60")
```

Comment:
```suggestion
		return nil, newInvalidSyntaxError("time of day", "minutes must be in the range from 0 to 60")
```

also maybe it would be nice to have this error message "error: "time of day" is invalid: minutes must be in the range from 0 to 60, got 'xx'"